tests/MEM/mem.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Enable -DTESTING in Makefile and the recompile library - make
```

Comment:
The relative paths here make me nervous.  Can you use the
git rev-parse --show-toplevel to find the top of the git tree
and use paths relative to that?